src/dotnet/APIView/APIView/App.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace ApiView
+{
+    public static class App
```

Comment:
Why not just use Program.cs?

Reply:
I swear it made sense at some point. Moved `RunAsync` to `Program`.